config/initializers/webauthn.rb

```diff
@@ -0,0 +1,4 @@
+WebAuthn.configure do |config|
+  config.origin = ENV.fetch("WEBAUTHN_ORIGIN", "http://localhost:3000")
```

Comment:
>  What is the impact of setting this config

This value needs to match `window.location.origin` during webauthn registration and authentication. So whatever the root_url is. However, since this is in a config file I can't use `root_url` like a could in a controller or view. 

I've found that we have our protocol and host setup in `Rails.application.config.rubygems.[protocol || host] so I'm going to use that to dynamically create the correct origin in Production and Staging, that way we don't need to rely on a ENV variable being set. I don't see a way to grab the current port though so I'll stick to the ENV + fallback to localhost::3000 for development. That way if a user is not running on localhost::3000 they can still set their own webauthn origin